doc/classes/FoldableGroup.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="FoldableGroup" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
For consistency, this should have the same search `keywords` as FoldableContainer does.